docs/wallets/spv.md

```diff
@@ -0,0 +1,60 @@
+# SPV (Simple Payment Verification)
+
+---
+
+## What is Simple Payment Verification?
+
+Simple Payment Verification (SPV) allows the use of a Decred wallet without having to download the entire Decred blockchain. A wallet operating in SPV mode is able to retrieve all of its transactions whilst only downloading blocks from the Decred network if they contained owned transactions. In a typical case a wallet will only need to download tens of megabytes rather than multiple gigabytes.
+
+SPV has been built directly into the `dcrwallet` CLI tool as a secondary operating mode. Decrediton uses `dcrwallet` behind the scenes, and as a result all users of Decred are able to enable SPV regardless of their wallet preference.
+
+
+## Why was SPV added to `dcrwallet`?
+
+The hardware requirements to run a Decred wallet are drastically reduced when operating in SPV mode. Storage and download requirements are reduced because rather than downloading the whole blockchain, an SPV wallet will only download the blocks which contain relevant transactions, and for all of the other blocks in the chain only the headers are downloaded. The amount of processing power required is reduced because a device running an SPV wallet will only validate the proof-of-work and the header chain rather than validating every transaction in every block and ensuring the block contents are consistent with the headers.
+
+As a result of these decreased requirements, Decred wallets can operate on a wider set of devices, particularly mobile devices. Smartphones and tablets are typically limited in at least one of CPU power, storage or download capacity, which makes running a full node either impractical or impossible. 
+
+Another benefit offered by SPV is an extreme reduction in the time required for a brand new wallet to become operational, offering a huge improvement to the user experience.
```

Comment:
mobile operating systems also greatly reduce the amount of background processing apps can perform, making running a full node unreliable and slow to use again after it was not running for a while.  The same problem can affect SPV wallets, but the sync to catch up is far faster.  This ties in with the previous paragraph about mobile app restrictions.  Not sure if you think this is important to detail here, but it is what it is.